base/server/python/pki/server/cli/banner.py

```diff
@@ -56,36 +71,19 @@ def usage(self):
 
     def execute(self, argv):
 
-        try:
-            opts, _ = getopt.gnu_getopt(argv, 'i:v', [
-                'instance=',
-                'verbose', 'debug', 'help'])
-
-        except getopt.GetoptError as e:
-            logger.error(e)
-            self.usage()
-            sys.exit(1)
+        args = self.parser.parse_args(args=argv)
 
-        instance_name = 'pki-tomcat'
+        if args.help:
+            self.print_help()
```

Comment:
The method `print_help()` is not defined in the BannerShowCLI.

The argparse has its way to print the help message, we could use it making easier to keep option and description aligned but not sure how to configure with these modular commands. Can be investigated in separate PR

Reply:
Good catch! I've renamed `usage()` to `print_help()`.

For now I just want to fix the deprecation warnings, but later we can try to use sub-parsers to define the options & help messages for the sub-commands.